src/app/taskBoard/TaskBoard.tsx

```diff
@@ -104,25 +109,45 @@ export const TaskBoard = () => {
     [lists, moveCard],
   )
 
+  const { showModal } = useSelector(selectCreateTask)
+
   return (
-    <Stack direction="row" columnGap={2} sx={{ padding: '0px 20px', width: '100vw' }}>
-      {Object.entries(lists).map(([listId, listData], index) => (
-        <Droppable
-          key={listId}
-          accept={'taskCard'}
-          index={index}
-          id={Number(listId)}
-          onDropItem={(item) => onDropItem(Number(listId), item)}
+    <AppMargin size={SizeofAppMargin.LARGE} ptb="18.5px">
+      <Stack
+        direction="row"
+        columnGap={2}
+        sx={{
+          overflowX: 'auto',
+        }}
+      >
+        {Object.entries(lists).map(([listId, listData], index) => (
+          <Droppable
+            key={listId}
+            accept={'taskCard'}
+            index={index}
+            id={Number(listId)}
+            onDropItem={(item) => onDropItem(Number(listId), item)}
+          >
+            <TaskColumn key={listId} columnName={`List ${listId}`} taskCount={String(listData.length)}>
+              {listData.map((task: Task, index: number) => (
+                <Droppable key={task.id} accept={'taskCard'} index={index} id={Number(listId)} moveCard={moveCard}>
+                  <TaskCard assignedTo={task.assignedTo} key={task.id} />
+                </Droppable>
+              ))}
+            </TaskColumn>
+          </Droppable>
```

Comment:
Maybe we can move this into it's own component - this should help modularize things

Reply:
`Droppable`, `TaskCard` and `TaskColumn` are already highly reusable component which can be used throughout the app. Moving this part of the code to a different component will increase complications more because we have to perform unnecessary prop drilling.